### DISCUSSION TITLE
Q&A / 问与答

### DISCUSSION THREAD

**Discussion by Xiaokang2022Vice** [Xiaokang2022/maliang-docs#25] [Q&A] [answered]
**Q1:** *What is the difference between this repository and the [main repository](https://github.com/Xiaokang2022/tkintertools)?*  
该仓库和[主仓库](https://github.com/Xiaokang2022/tkintertools)的区别是什么？

**Q2:** *Why separate the documentation repository from the source code repository?*  
为什么要将文档仓库和源代码仓库分开？

**Q3:** *How can the documentation repository help with learning about this project?*
文档仓库对于学习该项目有什么帮助？

**Q4:** *How to urge the author to update quickly?*  
如何催更？

**Comment by Xiaokang2022** [comment, marked as answer]
**A1:** *This repository is dedicated to storing related content for documentation, while the [main repository](https://github.com/Xiaokang2022/tkintertools) is used to store related content for the source code.*  
此仓库专门用于存储文档的相关内容，而[主仓库](https://github.com/Xiaokang2022/tkintertools)用于存储源代码的相关内容。

**A2:** *The document repository contains a lot of non-code resources, such as images, which will take up a lot of space and are not easy to manage in the main repository.*  
文档仓库中包含非常多的非代码资源，如图片等，存储在主仓库中会占用大量空间，不便于管理。

**A3:** *If you don't want to contribute documentation content or modify the style of the website, then you don't need to look at the source code content of the website, just [visit the website](https://xiaokang2022.github.io/tkintertools-docs/) to learn.*  
如果不是要贡献文档内容或者修改网站样式，那么就不需要看网站的源代码内容，直接[访问网站](https://xiaokang2022.github.io/tkintertools-docs/)学习即可。

**A4:** *#?$@/&,6...*
歪比巴卜……

**Comment by wumingshiali** [comment]


**Reply by Xiaokang2022** [reply]
请勿在本讨论下回复内容或提出其他问题，此讨论是给大家看的。

请[打开一个讨论](https://github.com/Xiaokang2022/tkintertools-docs/discussions/new?category=q-a)（该类型为Q&A）来进行提问，谢谢！
